data/mods/gen6/random-teams.ts

```diff
@@ -552,9 +549,9 @@ export class RandomGen6Teams extends RandomGen7Teams {
 				// Handle Hidden Power IVs
 				if (moveid === 'hiddenpower') {
 					const HPivs = this.dex.getType(move.type).HPivs;
-					for (const iv in HPivs) {
-						// @ts-ignore
-						ivs[iv] = HPivs[iv];
+					let iv: StatName;
+					for (iv in HPivs) {
```

Comment:
The main problem with this workaround is that it doesn't work in TypeScript 4.0 anymore. What's our plan there?

Reply:
Sorry, I was under the impression that this was our current workaround, so I just copied it here.

Reply:
Oh, it's fine. You don't need to fix this now. I was just wondering if you had any ideas for what we should do after TS 4.0.